.editorconfig

```diff
@@ -1,11 +1,98 @@
 root = true
 
-[**/*.cs]
+[**/*.xaml]
 indent_style = tab
 indent_size = 4
 tab_width = 4
 
-[**/*.xaml]
+[**/*.cs]
 indent_style = tab
 indent_size = 4
-tab_width = 4
\ No newline at end of file
+tab_width = 4
+
+# https://files.community/docs/contributing/code-style
+
+dotnet_naming_rule.pascal_case_for_classes.symbols = classes
+dotnet_naming_symbols.classes.applicable_kinds = class
+dotnet_naming_symbols.classes.applicable_accessibilities = *
+dotnet_naming_symbols.classes.required_prefix = 
+dotnet_naming_symbols.classes.required_suffix = 
+
+dotnet_naming_rule.pascal_case_for_interfaces.symbols = interfaces
+dotnet_naming_symbols.interfaces.applicable_kinds = interface
+dotnet_naming_symbols.interfaces.applicable_accessibilities = *
+dotnet_naming_symbols.interfaces.required_prefix = I
+dotnet_naming_symbols.interfaces.required_suffix = 
+
+dotnet_naming_rule.pascal_case_for_structs.symbols = structs
+dotnet_naming_symbols.structs.applicable_kinds = struct
+dotnet_naming_symbols.structs.applicable_accessibilities = *
+dotnet_naming_symbols.structs.required_prefix = 
+dotnet_naming_symbols.structs.required_suffix = 
+
+dotnet_naming_rule.pascal_case_for_methods.symbols = methods
+dotnet_naming_symbols.methods.applicable_kinds = method
+dotnet_naming_symbols.methods.applicable_accessibilities = *
+dotnet_naming_symbols.methods.required_prefix = 
+dotnet_naming_symbols.methods.required_suffix = Async
+
+dotnet_naming_rule.camel_case_for_variables.symbols = variables
+dotnet_naming_symbols.variables.applicable_kinds = field
+dotnet_naming_symbols.variables.applicable_accessibilities = private
+dotnet_naming_symbols.variables.required_prefix = _
+dotnet_naming_symbols.variables.required_suffix = 
+
+dotnet_naming_rule.camel_case_for_parameters.symbols = parameters
+dotnet_naming_symbols.parameters.applicable_kinds = parameter
+dotnet_naming_symbols.parameters.applicable_accessibilities = *
+dotnet_naming_symbols.parameters.required_prefix = 
+dotnet_naming_symbols.parameters.required_suffix = 
+
+dotnet_naming_rule.pascal_case_for_properties.symbols = properties
+dotnet_naming_symbols.properties.applicable_kinds = property
+dotnet_naming_symbols.properties.applicable_accessibilities = *
+dotnet_naming_symbols.properties.required_prefix = 
+dotnet_naming_symbols.properties.required_suffix = 
+
+dotnet_naming_rule.upper_case_for_constants.symbols = constants
+dotnet_naming_symbols.constants.applicable_kinds = field
+dotnet_naming_symbols.constants.applicable_accessibilities = *
+dotnet_naming_symbols.constants.required_prefix = 
+dotnet_naming_symbols.constants.required_suffix = 
+dotnet_naming_symbols.constants.required_capitalization = all_upper
```

Comment:
I think Pascal is fine. Title case is used in C/C++

Reply:
Wait, we require this. The normal C# convention is to use PascalCase for constants, we should amend.

Reply:
Sorry, you mean use PascalCase here? I don't understand

![image](https://github.com/user-attachments/assets/05cd53bd-70aa-43da-9abd-4a7f56c818b3)

The docs literally say use UPPER for constant variables.